include/clang/AST/ASTImporter.h

```diff
@@ -311,8 +311,10 @@ namespace clang {
     ///
     /// Subclasses can override this function to observe all of the \c From ->
     /// \c To declaration mappings as they are imported.
-    virtual Decl *Imported(Decl *From, Decl *To);
-      
+    virtual Decl *NotifyWhenImported(Decl *From, Decl *To) { return To; }
+
+    Decl *Imported(Decl *From, Decl *To);
+
```

Comment:
Maybe rename this to `MapImported` or something similar?

Reply:
`MarkAsImported`?